pages/login/index.js

```diff
@@ -120,7 +147,7 @@ function setApiKey(apiKey) {
   if (result == null)
     addBanner('Service worker did not acknowledge', true)
   else if (result === true || result['success'] === true)
-    onSuccessRedirectBasedOfPageWidth()
+    submitTrackerInformationAndRedirect(apiKey)
```

Comment:
this is fine but if you want your functions to be slightly more atomic this seems like a natural candidate for promise chaining, e.g. you can get the same thing from `submitTrackerInformation().then(() => redirectBasedOfPageWidth())` but then your functions each do one thing